spec.md

```diff
@@ -1457,6 +1443,7 @@ Supervised plugins MAY be isolated and/or resource-bounded.
 ##### Available Services
 
 * Plugin Packages MAY support all or a subset of CSI services; service combinations MAY be configurable at runtime by the Plugin Supervisor.
+  * A plugin must know the "mode" in which it's operating: mode of operation is NOT discoverable via interaction with the CO via the CSI specification.
```

Comment:
How about: `A plugin must know the "mode" in which it's operating (e.g. node, controller, or both). This specification does not dictate the mechanism by which mode of operation must be discovered, leaving it to the SP to decide.`